### PR TITLE
Fulfill support of PHP 7.4 features

### PR DESCRIPTION
**Pull request by kylekatarnls**
This PR fulfill the support of every PHP features up to PHP 7.4 on the top of the following remaining PRs:
- #406
- #408
- #409
- #410
- #411
- #412
- #413
- #415
- #421
- #422
- #426

### CODE REVIEW

**Comment by Bomberis** [comment]
Hurry it up!

**Comment by grifinas** [comment]
Fucking hurry it up already!

**Comment by kylekatarnls** [comment]
This is fucking impolite.

**Comment by kylekatarnls** [comment]
If you want this open-source library to be released faster, we're waiting for your contribution guys to help fixing bugs and and features.

File: src/main/php/PDepend/Source/Language/PHP/PHPParserVersion74.php [hunk 1]
```diff
@@ -119,4 +109,8 @@ protected function parseLambdaFunctionDeclaration()
 
         return $this->setNodePositionsAndReturn($closure);
     }
+
+    protected function checkEllipsisInExpressionSupport()
+    {
```
Comment (tvbeek) [inline src/main/php/PDepend/Source/Language/PHP/PHPParserVersion74.php]: Maybe it is good to add a comment that this function is intentional empty. It took me multiple times to verify that this is correct.

Comment (kylekatarnls) [reply src/main/php/PDepend/Source/Language/PHP/PHPParserVersion74.php]: Thanks. Done.

**Pull request merged**